executors/dart/pubspec.yaml

```diff
@@ -2,17 +2,17 @@ name: dart_executor
 publish_to: none
 
 environment:
-  sdk: ^3.7.0
+  sdk: ^3.10.0-37.0.dev
 
 # Add regular dependencies here.
 dependencies:
   collection: ^1.19.1
-  intl4x: ^0.11.3
+  intl4x: ^0.12.2
   node_preamble: ^2.0.2
   pubspec_lock_parse: ^2.2.0
```

Comment:
@mosuem Can you pin these versions (not with `^`) to reduce the risk of breakages? (for `intl4x` and `sdk`)'

Or, can we guarantee that we always install form the lockfile?